include/seqan3/alignment/pairwise/align_pairwise.hpp

```diff
@@ -162,36 +164,29 @@ constexpr auto align_pairwise(sequence_t && sequences,
     // Pipe with views::persist to allow rvalue non-view ranges.
     auto seq_view = std::forward<sequence_t>(sequences) | views::persist;
     // Configure the alignment algorithm.
-    auto kernel = detail::alignment_configurator::configure<decltype(seq_view)>(config);
+    auto && [algorithm, adapted_config] = detail::alignment_configurator::configure<decltype(seq_view)>(config);
 
     //!brief Lambda function to translate specified parallel and or vectorised configurations into their execution rules.
     constexpr auto get_execution_rule = [] ()
     {
-        using exec_rule = std::remove_reference_t<alignment_config_t>;
-
-        if constexpr (exec_rule::template exists<align_cfg::parallel>())
-        {
-            if constexpr (exec_rule::template exists<detail::vectorise_tag>())
-            {
-                return seqan3::par_unseq;
```

Comment:
Because we don't have a special excecution handler for vectorised code. We simply do the chunking as a high level functionality in the alignment executor. Only the algorithm that receives the chunks does the vectorisation or not.